src/components/Hello.js

```diff
@@ -0,0 +1,10 @@
+import { Component } from "react";
+import React from "react";
```

Comment:
you can put theses imports together on the same line to save a bit of redundant code